### PR TITLE
Calculator UI

### PR DESCRIPTION
**Pull request by meghaniankov**
# Description

* Update form and button styling for calculator variant
* Add calculator modal styling

image

# Checklist

Pull request contains:

- [ ] A new component
- [x] Component maintenance: improvement / bug fix / etc
- [ ] Component library change: storybook / webpack / etc

Definition of done:

- [x] Includes theme changes for both Uswitch and money
- [x] Work has been tested in multiple browsers
- [x] PR description includes description of change
- [x] PR description includes screenshot of change
- [ ] If new component, designer has approved screenshot
- [ ] If the change will affect other teams, that team knows about this change
- [ ] If introducing a new component behaviour, added a story to cover that case.

### CODE REVIEW

File: src/themes/save-on-energy/theme.json [hunk 1]
```diff
@@ -695,10 +695,33 @@
...
           "mt": "sm"
+        },
+        "variants": {
+          "calculator": {
+            "variant": "styles.h6",
+            "fontWeight": "bold",
+            "color": "grey-80",
+            "display": "block",
+            "mb": "xs",
+            ":not(:first-child)": {
```
Comment (Segger87) [inline src/themes/save-on-energy/theme.json:706]: have you browser checked this? https://caniuse.com/#search=%3Anot() I dont think not is well supported at all

Comment (Segger87) [reply src/themes/save-on-energy/theme.json:706]: might be fine actually as your using it as a negation, but just want to check

Comment (meghaniankov) [reply src/themes/save-on-energy/theme.json:706]: It does actually appear to be fine in IE11, not sure why caniuse is saying it wouldnt

Comment (barneyfoxuk) [reply src/themes/save-on-energy/theme.json:706]: yeah I would never use `:not`. You should be able to do the same thing by giving `calculator` the `"mt": "xl"` and then doing `:first-child: { mt: 0 }`

**Pull request merged**